src/RulesContainer.js

```diff
@@ -131,6 +133,17 @@ export default class RulesContainer {
   }
 
   /**
+   * Link renderable rules with CSSRuleList.
+   */
+  link(cssRules: CSSRuleList): void {
+    for (let i = 0; i < cssRules.length; i++) {
+      const CSSStyleRule = cssRules[i]
```

Comment:
I am a bit wondering why we call a variable like this) I think I started to do that but its not really nice convention-wise, maybe simple `cssRule` is better. Also the idea to call them `renderable` was in order to abstract away from cssom/dom terminology because everything dom related should be in DomRenderer